src/ui/components.tsx

```diff
@@ -77,7 +77,13 @@ export const CheckMkAsyncSelect = function <Key extends keyof RequestSpec, Value
         },
         (error) => {
           if (error && error.message) {
-            setAutocompleteError(error.message);
+            if (error.message === "Sorry, you cannot create combined graphs for more than 100 objects") {
+              setAutocompleteError(
+                "You reached the query size limit! Please add a few more filters to narrow the results. "
```

Comment:
```suggestion
                "You reached the query size limit. Add more filters to reduce the number of elements."
```
maybe we should also get rid of the "You" to make it sound more informal?